solutions/instances/variables.tf

```diff
@@ -111,15 +223,21 @@ variable "enable_platform_logs" {
 variable "enable_at_event_routing_to_cos_bucket" {
   type        = bool
   description = "Whether to enable event routing from Activity Tracker to the Object Storage bucket."
-  default     = true
+  default     = false
```

Comment:
After internal discussions, we decided to keep this as true aswell as `enable_at_event_routing_to_cloud_logs`.